### PR TITLE
T012 Firebase

### PR DESCRIPTION
**Pull request by MaxDangVu136**
Issue: Needed to put Max's google signup implementation in a separate folder from working Firebase code. 

Solution: Added files to folder 'google_signup_unfinished' for potential use later, also commented briefly on what each file does. 

Risk: Minimal

### CODE REVIEW

File: src/components/Authentication/SignIn.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    // sends details to firebase sign in method
+    const handleSubmit = (event: any) => {
+        // prevents email and password to show up on URL
+        event.preventDefault();
+        
+        // calls sign in via email
+        SignInViaEmail(email, password);
+
+    }
+
```
Comment (KennethZhu904) [inline src/components/Authentication/SignIn.tsx:30]: remove that space

**Review by KennethZhu904 — APPROVED** [review]
LGTM. In the future, if you are making notes to remind yourself what the code does, please make sure to remove them . I suggest you put a word that prompt you to remove your comment once you finish the task. Some comments in your code are not neccessary. E.g. line 17 and 20 of Admin.tsx file as it is understandable from developers point of view.

**Review by johnchen383 — CHANGES_REQUESTED** [review]
Could we add Admin.tsx to the screens folder. Also, nitpicking but can we change Authentication to lowercase for consistency. Also FirebaseConfig.

**Review by johnchen383 — CHANGES_REQUESTED** [review]
Can we delete package-lock.json? And ensure a rebuild with `yarn install` then `yarn start`. Sorry shouldve spotted that first time round.

**Review by johnchen383 — APPROVED** [review]
LGTM. Feel free to merge and move ur notion ticket

**Pull request merged**